src/create_oauth2_client.ts

```diff
@@ -169,6 +169,41 @@ export function createSlackOAuth2Client(
   });
 }
 
+/**
+ * Creates an OAuth 2.0 client with Twitch as the provider.
+ *
+ * Requires environment variables:
+ * 1. TWITCH_CLIENT_ID
+ * 2. TWITCH_CLIENT_SECRET
+ *
+ * ```ts
+ * import { createTwitchOAuth2Client } from "https://deno.land/x/deno_kv_oauth/mod.ts";
+ *
+ * const oauth2Client = createTwitchOAuth2Client({
+ *  redirectUri: "http://localhost:8000/callback",
+ *  defaults: {
+ *    scope: "users.read"
```

Comment:
Is this scope valid for this given OAuth2 client?

Reply:
Oh shoot, I forgot to change that. Apologies.